libs/@local/harpc/net/src/session/server/test.rs

```diff
@@ -225,7 +225,13 @@ impl EchoService {
 
     fn spawn(self, mut server: ListenStream) {
         tokio::spawn(async move {
-            while let Some(transaction) = server.next().await {
+            // you may ask yourself: why not `while let`?
+            //  significant drop in scrutiny clippy warning.
+            loop {
```

Comment:
```suggestion
            // Using `loop` instead of `while let` to avoid significant drop in scrutiny
            loop {
```

Reply:
got too sassy in my explanation comment D:

Reply:
🤏🏻 😄